src/overlays/actors/ovl_Fishing/z_fishing.h

```diff
@@ -9,28 +9,26 @@ struct Fishing;
 typedef struct Fishing {
     /* 0x0000 */ Actor actor;
     /* 0x014C */ char unk_14C[0x004];
-    /* 0x0150 */ u8 unk_150;
-    /* 0x0151 */ u8 unk_151;
+    /* 0x0150 */ u8 isLoach; //rename to fish type?
+    /* 0x0151 */ u8 unk_151; //lily byte
     /* 0x0152 */ u8 unk_152;
     /* 0x0154 */ s16 unk_154;
     /* 0x0156 */ u8 unk_156;
     /* 0x0157 */ u8 unk_157;
     /* 0x0158 */ s16 unk_158;
     /* 0x015A */ s16 unk_15A;
-    /* 0x015C */ s16 unk_15C;
+    /* 0x015C */ s16 state; //fish use as bitfeild, owner as talking state
     /* 0x015E */ s16 unk_15E;
-    /* 0x0160 */ s16 unk_160;
-    /* 0x0162 */ s16 unk_162;
-    /* 0x0164 */ s16 unk_164;
-    /* 0x0166 */ s16 unk_166;
-    /* 0x0168 */ s16 unk_168;
-    /* 0x016A */ s16 unk_16A;
-    /* 0x016C */ s16 unk_16C;
+    /* 0x0160 */ s16 unk_160; //fish use as rotateX, owner as index of eye texture
+    /* 0x0162 */ s16 unk_162; //fish use as rotateY, owner as index of eye texture
+    /* 0x0164 */ s16 unk_164; //fish use as rotateZ, owner as rotation of head
+    /* 0x0166 */ Vec3s rotationStep;
+    /* 0x016C */ s16 limb23RotYDelta;
     /* 0x016E */ s16 unk_16E;
     /* 0x0170 */ s16 unk_170;
-    /* 0x0172 */ s16 unk_172;
-    /* 0x0174 */ s16 unk_174;
-    /* 0x0176 */ s16 unk_176;
+    /* 0x0172 */ s16 limbEFRotYDelta;
+    /* 0x0174 */ s16 limb89RotYDelta;
+    /* 0x0176 */ s16 limb4RotYDelta;
```

Comment:
Maybe prefix these and `limb23RotYDelta` with "fish" to differentiate it from all the other things in this actor?